packages/spl/src/claimable-tokens/constants.ts

```diff
@@ -2,3 +2,30 @@ export enum ClaimableTokensInstruction {
   Create = 0,
   Transfer = 1
 }
+
+/**
+ * Custom error codes from the Claimable Tokens program
+ * @see {@link https://github.com/AudiusProject/audius-protocol/blob/2a37bcff1bb1a82efdf187d1723b3457dc0dcb9b/solana-programs/claimable-tokens/program/src/error.rs solana-programs/claimable-tokens/program/src/error.rs}
+ */
+export enum ClaimableTokensErrorCode {
+  SignatureVerificationFailed = 0,
```

Comment:
nit: why does the first one have `= 0` but not the others

Reply:
because it increments from there, this is an enum that starts counting at 0